From 9c01dab556df49aedf63c83799f33a76af7dbabb Mon Sep 17 00:00:00 2001
From: Herman Chen <herman.chen@rock-chips.com>
Date: Thu, 22 Oct 2020 18:20:25 +0800
Subject: [PATCH] video: rockchip: mpp:  Handle double quote in log

When there is double quote in last git log mpp Makefile will report
error. Add more subst to handle the double quote.

Signed-off-by: Herman Chen <herman.chen@rock-chips.com>
Change-Id: Ifa34a89cf917a5837987e5bbf61def9ed37a2f4f
---
 drivers/video/rockchip/mpp/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/mpp/Makefile b/drivers/video/rockchip/mpp/Makefile
index e98b6c330cd9..f497a082dc2a 100644
--- a/drivers/video/rockchip/mpp/Makefile
+++ b/drivers/video/rockchip/mpp/Makefile
@@ -5,7 +5,8 @@ MPP_GIT_REVISION := \
 	--pretty=format:"%h author: %<|(30)%an %cd %s" -- $(src) || \
 	echo -n "unknown mpp version for missing VCS info")
 
-MPP_REVISION := $(subst \,\\\,$(MPP_GIT_REVISION))
+MPP_REVISION_0 := $(subst \,\\\,$(MPP_GIT_REVISION))
+MPP_REVISION   := $(subst ",\\\",$(MPP_REVISION_0))
 
 rk_vcodec-objs := mpp_service.o mpp_common.o mpp_iommu.o
 CFLAGS_mpp_service.o += -DMPP_VERSION="\"$(MPP_REVISION)\""
-- 
2.35.3

